CHANGELOG.md

```diff
@@ -16,6 +16,23 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   pods, would trigger events with warnings. See this
   [bug](https://bugzilla.redhat.com/show_bug.cgi?id=2088202) for more
   information. No action is required from users to consume this fix.
+- Fixes to OCP4/RHCOS4 compliance content:
```

Comment:
If we want, we can make this a separate section

Reply:
Would we have a separate section for just content fixes, or content in general?